ballerina/xmldata.bal

```diff
@@ -43,6 +43,10 @@ public type JsonOptions record {
 # + return - XML representation of the given JSON if the JSON is
 # successfully converted or else an `xmldata:Error`
 public isolated function fromJson(json jsonValue, JsonOptions options = {}) returns xml?|Error {
+    if (jsonValue is string || jsonValue is boolean || jsonValue is decimal || jsonValue is float ||
```

Comment:
Shall we move this to https://github.com/ballerina-platform/module-ballerina-xmldata/blob/5b9b7bee4099e8c1b9008c9a94015d1fb79f16a8/ballerina/xmldata.bal#L67 (else block) and change it like

```
if !(jsonValue is null) {
    return xml:createText(jsonValue.toString());
}
```

Reply:
Can't move to Line 67, so I moved that to [here](https://github.com/ballerina-platform/module-ballerina-xmldata/pull/359/files#diff-ecda4f4acb566335c5b8a6598ba4ffc0002c3cf114c6867b20c16bb26d67780bR70). But, IMO, If JSON has a single value, we can return a text value without [these checks](https://github.com/ballerina-platform/module-ballerina-xmldata/pull/359/commits/89e4aae074d0190cb2410f9d1ea94e5285a0195e#diff-ecda4f4acb566335c5b8a6598ba4ffc0002c3cf114c6867b20c16bb26d67780bR46). So I added in the first.